python/paddle/fluid/install_check.py

```diff
@@ -24,11 +25,35 @@
 from . import compiler
 import logging
 import numpy as np
-import os
 
 __all__ = ['run_check']
 
 
+def process_env():
+    env = os.environ
+    device_list = []
+    if env.get('CUDA_VISIBLE_DEVICES') is not None:
+        cuda_devices = env['CUDA_VISIBLE_DEVICES']
+        if core.get_cuda_device_count() == 0:
+            logging.warning(
+                "No CUDA Device Found! But your are using GPU version Paddle Fluid"
+            )
+            device_list = []
+        elif len(cuda_devices) == 1:
```

Comment:
What if there are more than 10 cards in one machine. It may be `CUDA_VISIBLE?_DEVICES=10`.

Reply:
This function can be simplify by `return list(range(core.get_cuda_device_count()))`.